Mejorando el flujo de Arquitectura Hexagonal

RevillaDev opened the pull request:
<!-- 
Thanks for creating this pull request 🤗

Please make sure that the pull request is limited to one type (docs, feature, etc.) and keep it as small as possible. You can open multiple PRs instead of opening a huge one.
-->

## 📑 Description
<!-- Add a brief description of the pr -->
Mejorando el flujo de la Arquitectura.

<!-- You can also choose to add a list of changes and if they have been completed or not by using the markdown to-do list syntax
- [x] Completed
- [ ] Not Completed
-->

## ✅ Checks
<!-- Make sure your pr passes the CI checks and do check the following fields as needed - -->
- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

## ℹ Additional Information
<!-- Any additional information like breaking changes, dependencies added, screenshots, comparisons between new and old behavior, etc. -->

RevillaDev committed:
Mejorando el flujo de Arquitectura Hexagonal

RevillaDev committed:
Se modifica NotificacionDto a Notificacion en FeedbackService

JamiltonQuintero commented on src/main/java/com/entrevistador/generadorfeedback/domain/model/dto/NotificacionDto.java:
```diff
@@ -0,0 +1,22 @@
+@NoArgsConstructor
+@Builder
+public class NotificacionDto {
+    @JsonProperty("tipo")
```
Kire quitemos las notaciones de los dominios porfavor me refiero a las que no sean lombok

JamiltonQuintero replied:
Y la otra es pro quedó un dto en el dominio?

JamiltonQuintero commented on src/main/java/com/entrevistador/generadorfeedback/domain/port/FeedbackDao.java:
```diff
@@ -1,19 +1,22 @@
-    Mono<FeedbackDto> actualizarFeedback(FeedbackDto feedbackComentarioDto);
-    Flux<PreguntaComentarioDto> obtenerPreguntas(String entrevistaId);
-    Flux<FeedbackResponseDto> obtenerFeedback(String entrevistaId);
+    Mono<Respuesta> actualizarRespuestas(Respuesta respuestaDto);
```
Podemos realizar limpieza decampos de renombrados ya que confunde un poco que sea dominio pero aun diga Dtos (en general en todo el refactor)

RevillaDev committed:
Moviendo domain.model.dto a infrastructure.adapter.dto
- Modificando nombres de argumentos donde se utilizaban los dtos
- Formateando el codigo

Kirenai commented:
Modificación realizada.

Kirenai commented:
upps

JamiltonQuintero commented:
Cerrado. Gracias Kire
